.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+
+# Thanks for contributing a pull request! 
+## Please make sure you see our contribution guidelines: https://github.com/narwhals-dev/narwhals/blob/main/CONTRIBUTING.md
+
+## What type of PR is this? (check all applicable)
+
+- [ ] 💾 Refactor
+- [ ] ✨ Feature
+- [ ] 🐛 Bug Fix
+- [ ] 🔧 Optimization
+- [ ] 📝 Documentation
+- [ ] ✅ Test
+- [ ] 🐳 Other
+
+## Related Issues 
+
+- Related Issue # 
+- Closes #
+
+## About tests
+
+- [ ] Tests added. If you are adding a new feature, please also add a test for it. 
+- [ ] No, I did not add a test.
```

Comment:
I know the cicd would fail otherwise, but on a more broad spectrum we could ask also for linting and documenting. Maybe:

```suggestion
## Checklist

- [ ] Code follows style guide (ruff)
- [ ] Tests added
- [ ] Documentation changes
```

Reply:
Done, thanks